src/pg-quote.vue

```diff
@@ -1,24 +1,52 @@
 <template>
   <div class="paragraph paragraph--quote">
-    <div 
-      class="paragraph__field-quote" 
-      v-html="fieldText"
-      v-if="hasText"
-    >
-    </div>
-    <slot name="field-text"></slot>
+    <blockquote class="content">
+      <header
+        v-if="$slots.title"
+        class="title"
+      >
+        <slot name="title" />
+      </header>
+      <section class="quote">
+        <slot :name="$slots.quote ? 'quote': 'default'" />
+      </section>
+      <footer
+        v-if="$slots.author"
+        class="author"
+      >
+        <slot name="author" />
+      </footer>
+    </blockquote>
   </div>
 </template>
 <script>
   export default {
-    name: 'PgQuote',
-    props: {
-      dataFieldText: { type: String, default: () => '' },
-    },
-    computed: {
-      hasText() {
-        return this.dataFieldText !== '';
-      },
-    }
+    name: 'PgQuote'
   };
 </script>
+<style lang="scss" scoped>
+.paragraph--quote {
+  .content {
+    padding: 1em 2em;
+    border-left: 3px solid grey;
```

Comment:
what if someone does not want this styles? Maybe add a .style class and a prop to remove it via style=false or so?